src/ert/shared/services/webviz_ert_service.py

```diff
@@ -1,20 +1,19 @@
 import sys
+from typing import Any
 
 from ert.shared.services._base_service import BaseService
 
 
 class WebvizErt(BaseService):
     service_name = "webviz-ert"
 
-    def __init__(
-        self, title: str = None, experimental_mode: bool = False, verbose: bool = False
-    ):
+    def __init__(self, **kwargs: Any):
         exec_args = [sys.executable, "-m", "webviz_ert"]
-        if experimental_mode:
+        if kwargs.get("experimental_mode", False):
             exec_args.append("--experimental-mode")
-        if verbose:
+        if kwargs.get("verbose", False):
             exec_args.append("--verbose")
-        if title:
-            exec_args.append("--title")
-            exec_args.append(title)
-        super().__init__(exec_args)
+        exec_args.extend(["--title", str(kwargs.get("title"))])
+        exec_args.extend(["--project_identifier", str(kwargs.get("project"))])
+
+        super().__init__(exec_args, project=kwargs.get("project"))
```

Comment:
just to be a bit nitpicky - `kwargs.get("project")` can be refactored for DRYness :nerd_face: